hydromt/io/path.py

```diff
@@ -21,13 +21,13 @@ def _relpath(value, root):
     # loop through n-level of dict
     for key, val in cfdict.items():
         if isinstance(val, dict):
-            cfdict[key] = parse_relpath(val, root)
+            cfdict[key] = make_config_paths_relative(val, root)
         else:
             cfdict[key] = _relpath(val, root)
     return cfdict
 
 
-def parse_abspath(
```

Comment:
Is there a test for this function?

Reply:
not directly but it's included in https://github.com/Deltares/hydromt/blob/84aafa49bc5f73359d5893da610e33e8931956d7/tests/models/test_config.py#L10 
I can make a separate one if you like